packages/auth/src/server/middleware/callback.ts

```diff
@@ -31,7 +28,7 @@ export const callbackHandler: RouteHandler = async ({
       userID: session.user_id,
     }));
 
-  const redirection = NextResponse.redirect(config.appUrl(redirectUri));
+  const redirection = NextResponse.redirect(config.appUrl(redirectUri), 301);
```

Comment:
コメントが欲しいですので、以下でお願いします。

```suggestion
  // Next.js server redirection uses 307 and 308 to preserve the request method.
  // However the auth package always has to redirect users to the destination pages, so here intentionally uses 301 to make it GET request.
  const redirection = NextResponse.redirect(config.appUrl(redirectUri), 301);
```